tests.py

```diff
@@ -126,17 +127,16 @@ def test_make_cff(self):
 
     def test_run_makefakedata(self):
         self.Writer.make_data(verbose=True)
-        numSFTs = int(np.ceil(self.duration / self.Tsft))
```

Comment:
This and the next 2 diffs looks like a rebase accident...? This was just introducede in #205.

Reply:
I'm equally surprised. I'll fix it.